app/templates/components/addon-tabs.hbs

```diff
@@ -28,53 +28,52 @@
     >
       Deploy
     </AddonTabs::Tab>
+
   </div>
 
   {{!-- template-lint-disable block-indentation  --}}
   <AddonTabs::Panel
     @currentTab={{this.currentTab}}
     @tabId={{0}}
   >
-    <p>Ember CLI Sass wires up your app with Sass, in a single command.</p>
-
+    <p>Ember Concurrency is an Ember addon that makes state management a whole lot easier with useful primitives.</p>
```

Comment:
Same remark here. Maybe we can simply use the description from the [website](http://ember-concurrency.com/docs/introduction/)?

> ember-concurrency is an Ember Addon that makes it easy to write concise, robust, and beautiful asynchronous code